scss/_settings_font.scss

```diff
@@ -1,6 +1,6 @@
 // Global font settings
-$font-base-family: '"Ubuntu", -apple-system, "Segoe UI", "Roboto", "Oxygen", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif' !default;
-$font-monospace: '"Ubuntu Mono", Consolas, Monaco, Courier, monospace' !default;
+$font-base-family: 'Ubuntu', -apple-system, 'Segoe UI', 'Roboto', 'Oxygen', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif !default;
+$font-monospace: 'Ubuntu Mono', consolas, monaco, courier, monospace !default;
```

Comment:
property values are case-insensitive, but if we're changing to lowercase maybe we should change all? Or leave all uppercase as this change isn't really needed.

Reply:
Linter was complaining about upper case values, so that's why the change was made.
I guess uppercase is fine in case of names in quotes ('Ubuntu Mono', 'Roboto'), so we can put them all in quotes (uppercase) or remove quotes from single word ones and lowercase them too, still, we will need quotes around 'Ubuntu Mono' etc - not sure if they also are case insensitive? I guess they should...